evaluate.py

```diff
@@ -33,14 +33,12 @@ def main(args) -> None:
 
     # prepare dataloader
     pipeline = DataPipeline(pipline_config=config.pipeline, preprocessor_config=config.preprocessor)
-    pipeline.setup()
 
-    train_dataloader = pipeline.get_dataloader(pipeline.train_dataset, shuffle=False, drop_last=False,
+    dataset = pipeline.get_dataset(pipeline.dataset_builder,
+                                   getattr(config.pipeline.dataset.path, args.type),
```

Comment:
`getattr(config.pipeline.dataset.path, args.type)` 이 부분을 따로 빼서 설명변수를 넣어주면 좋을 것 같습니다.
단순히 `getattr`과 `getattr`에 사용되는 클래스 타입과 parameter가 어떤 인스턴스를 리턴하는지 확인하기 어려운 것 같아요.
```
dataset_cls = getattr(config.pipeline.dataset.path, args.type)
dataset = pipeline.get_dataset(pipeline.dataset_builder, dataset_cls...)
```

Reply:
> `getattr(config.pipeline.dataset.path, args.type)` 이 부분을 따로 빼서 설명변수를 넣어주면 좋을 것 같습니다.
> 단순히 `getattr`과 `getattr`에 사용되는 클래스 타입과 parameter가 어떤 인스턴스를 리턴하는지 확인하기 어려운 것 같아요.
> 
> ```
> dataset_cls = getattr(config.pipeline.dataset.path, args.type)
> dataset = pipeline.get_dataset(pipeline.dataset_builder, dataset_cls...)
> ```

오 넵넵 반영하겠습니다. 마틴님이 기록해두신 부분은 사실상 `dataset_cls` -> `dataset_path`로 변경하면 될 것 같아요.

Reply:
네네! 제가 저 getattr이 어떤 의미인지 몰라서 임의로 적어두었습니다!

Reply:
저는 `getattr` 을 class 의 method 를 호출할 때만 사용하는 것을 보았어서 그런지, dict의 value를 불러온다고 생각하기 어려웠던 것 같아요. 아마 그래서 @ssaru 님도 `dataset_cls`라고 하신게 아닐까 싶은데요,

dictionary의 value를 받아오는 것이라면, 보다 이 내용이 드러날 수 있도록 아래와 같이 변경하는 것은 어떨까요?
```suggestion
                                   config.pipeline.dataset.path.get(args.type),
```
```suggestion
                                   config.pipeline.dataset.path[args.type],
```